pym/bob/cmds/build.py

```diff
@@ -41,6 +42,24 @@ def hashWorkspace(step):
     return hashDirectory(step.getWorkspacePath(),
         os.path.join(step.getWorkspacePath(), "..", "cache.bin"))
 
+class LocalBuilderStatistic:
+    def __init__(self):
+        self.__activeOverrides = []
+        self.checkouts = 0
+        self.packagesBuilt = 0
+        self.packagesDownloaded = 0
+
+    def addOverrides(self, overrides):
+        for o in overrides:
+            if not o in self.__activeOverrides:
+                self.__activeOverrides.append(o)
+
+    def getActiveOverrides(self):
+        return self.__activeOverrides
+
+    def __getitem__(self, item):
+        return getattr(self, item)
```

Comment:
The problem is `str(stats['checkouts'])`. Just change it to `str(stats.checkouts)` and it should IMHO work...